src/averageAges.js

```diff
@@ -14,13 +16,22 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  const menInCentury = men.filter(man => Math.ceil(man.died / 100) === century);
+
+  men = century ? menInCentury : men;
```

Comment:
let's move ternary to do it in your `const`
```suggestion
  const menInCentury = century
    ? men.filter(man => Math.ceil(man.died / 100) === century)
    : men;
```